java/src/Ice/src/main/java/com/zeroc/Ice/UnknownUserException.java

```diff
@@ -1,43 +1,20 @@
-//
-// Copyright (c) ZeroC, Inc. All rights reserved.
-//
-//
-// Ice version 3.7.10
-//
-// <auto-generated>
-//
-// Generated from file `LocalException.ice'
-//
-// Warning: do not edit this file.
-//
-// </auto-generated>
-//
+// Copyright (c) ZeroC, Inc.
 
 package com.zeroc.Ice;
 
 /**
- * An operation raised an incorrect user exception. This exception is raised if an operation raises
- * a user exception that is not declared in the exception's <code>throws</code> clause. Such
- * undeclared exceptions are not transmitted from the server to the client by the Ice protocol, but
- * instead the client just gets an {@link UnknownUserException}. This is necessary in order to not
- * violate the contract established by an operation's signature: Only local exceptions and user
- * exceptions declared in the <code>throws</code> clause can be raised.
+ * The dispatch returned a {@link UserException} that was not declared in the operation's exception
+ * specification (the <code>throws</code> clause). This is necessary in order to not violate the
+ * contract established by an operation's signature: Only local exceptions and user exceptions
+ * declared in the <code>throws</code> clause can be raised.
  */
-public class UnknownUserException extends UnknownException {
-  public UnknownUserException() {
-    super();
+public final class UnknownUserException extends UnknownException {
+  public UnknownUserException(String message) {
+    super(message);
   }
 
-  public UnknownUserException(Throwable cause) {
-    super(cause);
-  }
-
-  public UnknownUserException(String unknown) {
-    super(unknown);
-  }
-
-  public UnknownUserException(String unknown, Throwable cause) {
-    super(unknown, cause);
+  public UnknownUserException(String message, Throwable cause) {
```

Comment:
Please no Throwable.

Reply:
I removed this.
In Java, we used to store the actual `UserException` in this field, so we could put it's stack trace in the `expectedDetails`.
Removing this does means that the details will be less... detailed now.